lib/Dashboard/Model/Incidents.pm

```diff
@@ -256,9 +256,9 @@ sub _update ($self, $db, $incident) {
 
   $db->query(
     'UPDATE incidents SET packages = ?, rr_number = ?, review = ?, review_qam = ?, approved = ?, emu = ?, active = ?,
-       embargoed = ? WHERE id = ?', $incident->{packages}, $incident->{rr_number}, $incident->{inReview},
+       embargoed = ?, priority = ? WHERE id = ?', $incident->{packages}, $incident->{rr_number}, $incident->{inReview},
     $incident->{inReviewQAM}, $incident->{approved}, $incident->{emu}, $incident->{isActive}, $incident->{embargoed},
-    $id
+    $incident->{priority},    $id
```

Comment:
extra spaces?

Reply:
Don't worry about it, we let perltidy decide.

Reply:
Yes. In fact it was tidy's work (possibly to achieve alignment with the list element in the line above).